src/contracts/Test/IbexaKernelTestTrait.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Contracts\Core\Test;
+
+use Doctrine\DBAL\Connection;
+use eZ\Publish\API\Repository\ContentService;
+use eZ\Publish\API\Repository\ContentTypeService;
+use eZ\Publish\API\Repository\LanguageService;
+use eZ\Publish\API\Repository\LocationService;
+use eZ\Publish\API\Repository\ObjectStateService;
+use eZ\Publish\API\Repository\PermissionResolver;
+use eZ\Publish\API\Repository\RoleService;
+use eZ\Publish\API\Repository\SearchService;
+use eZ\Publish\API\Repository\SectionService;
+use eZ\Publish\API\Repository\Tests\LegacySchemaImporter;
+use eZ\Publish\API\Repository\UserService;
+use eZ\Publish\Core\Repository\Values\User\UserReference;
+use eZ\Publish\SPI\Persistence\TransactionHandler;
+use eZ\Publish\SPI\Tests\Persistence\FixtureImporter;
+use RuntimeException;
+
+/**
+ *  @experimental
+ */
+trait IbexaKernelTestTrait
+{
+    private static $anonymousUserId = 10;
+
+    private static $adminUserId = 10;
```

Comment:
maybe it would be better to make them public consts in IbexaTestKernel?

Reply:
I'd rather prefer them to stay here, because if we're gonna go with your suggestion about a bootstrapping service (which it would be effectively) then we would want IbexaTestKernel to contain as little custom code as possible.

We could maybe extract these from Kernel parameters, as I think we've discussed once that those are stored somewhere in ParametersBag?